selftests/unit/nrunner.py

```diff
@@ -321,7 +321,7 @@ def test_runner_python_unittest_skip(self):
         runner_klass = runnable.pick_runner_class()
         runner = runner_klass()
         results = [status for status in runner.run(runnable)]
-        if sys.version_info < (3, 12, 1):
+        if sys.version_info < (3, 12, 1) or sys.version_info >= (3, 12, 2):
```

Comment:
Maybe it is just a personal preference, but I would inverse this condition to `sys.version_info == (3, 12, 1)`. IMO, it will improve the code readability.